src/antsibull_docs/data/sphinx_init/requirements_txt.j2

```diff
@@ -2,7 +2,7 @@
 # GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
 # SPDX-License-Identifier: GPL-3.0-or-later
 
-antsibull-docs >= 1.0.0, < 2.0.0
+antsibull-docs >= 2.0.0, < 3.0.0
```

Comment:
The current version `2.0.0a1` isn't included in this range. I assume we'll create a pre-release before 2.0.0, so we don't want to generate requirements.txt files that can't be resolved. We can change this after we release 2.0.0.

```suggestion
antsibull-docs >= 2.0.0a1, < 3.0.0
```

Reply:
Fixed in 42b2d95658e889f0f389494a9b2bc7712446c3a1.